geopandas/geodataframe.py

```diff
@@ -8,8 +8,8 @@
 from shapely.geometry.base import BaseGeometry
 from six import string_types, PY3
 
-from geopandas import GeoSeries
 from geopandas.base import GeoPandasBase
+from geopandas.geoseries import GeoSeries, _CoordinateIndexer
```

Comment:
since it is now for both geodataframe as geoseries, it maybe makes sense to move `_CoordinateIndexer` to base.py ?